src/DotVVM.Framework/Routing/RedirectingPresenter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DotVVM.Framework.Hosting;
+
+namespace DotVVM.Framework.Routing
+{
+    internal class RedirectingPresenter : IDotvvmPresenter
```

Comment:
Change to `public`
Rename to `DelegatePresenter` since it's a generic presenter implementation